pymilvus/client/prepare.py

```diff
@@ -1461,6 +1461,30 @@ def operate_privilege_request(
             type=operate_privilege_type,
         )
 
+    @classmethod
+    def operate_privilege_v2_request(
+        cls,
+        role_name: str,
+        privilege: str,
+        operate_privilege_type: Any,
+        db_name: str,
+        collection_name: str,
+    ):
+        check_pass_param(role_name=role_name)
+        check_pass_param(privilege=privilege)
+        check_pass_param(operate_privilege_type=operate_privilege_type)
+        check_pass_param(db_name=db_name)
+        check_pass_param(collection_name=collection_name)
```

Comment:
this should be able to merged into one `check_pass_param(role_name=role_name, privilege=privilege, ....)`